app-devel/blueprint-compiler/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=blueprint-compiler
+PKGSEC=devel
+ABSPLITDBG=0
```

Comment:
There's a meson build option that compiles documentation (default is false), enable it to include docs in the package.